Makefile

```diff
@@ -866,6 +866,17 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint-build:
```

Comment:
If we go ahead with this approach, should we also have a target to update/reinstall `remark`, e.g., after jumping between the branches that depend on different versions of it? (This is one of the reasons why I'd prefer it to be bundled with `node_modules`, tbh.)